snap/README.md

```diff
@@ -0,0 +1,99 @@
+* [Kata Containers snap image](#kata-containers-snap-image)
```

Comment:
Thanks for adding! Minor nit - it looks a lot better I find if you put the TOC just below the title.